README.md

```diff
@@ -47,22 +83,27 @@ Since writing FHE programs can be difficult, Concrete, based on LLVM, make this
 | macOS 11+ (Apple Silicon: M1, M2, etc.)     |         Yes         |       Yes        |
 
 
+#### Docker
 The preferred way to install Concrete is through PyPI:
 
 ```shell
 pip install -U pip wheel setuptools
 pip install concrete-python
 ```
 
+#### Pip
 You can get the concrete-python docker image by pulling the latest docker image:
 
 ```shell
 docker pull zamafhe/concrete-python:v2.0.0
 ```
 
-You can find more detailed installation instructions in [installing.md](docs/getting-started/installing.md)
+*Find more detailed installation instructions in [this part of the documentation](https://docs.zama.ai/concrete/getting-started/installing)*
+<br></br>
 
-## Getting started
+### 🟨 A simple example
+To compute on encrypted data, you first need to define the function you want to compute, then compile it into a Concrete Circuit, which you can use to perform homomorphic evaluation.
+Here is the full example that we will walk through:
```

Comment:
we don't really "walk through" this example, I mean, we don't explain step by step